lib/pages/chat/chat.dart

```diff
@@ -1632,6 +1621,42 @@ class ChatController extends State<Chat>
     _currentChatScrollState = ChatScrollState.scrolling;
   }
 
+  void _requestInputFocus() {
+    if (!inputFocus.hasPrimaryFocus) {
+      inputFocus.requestFocus();
+    }
+  }
+
+  void _updateReplyEvent({Event? event}) {
+    try {
+      replyEventNotifier.value = event;
+    } on FlutterError catch (e) {
+      Logs().e(
+        'Chat::_updateReplyEvent():: FlutterError: $e',
+        e.stackTrace,
+      );
+    }
+  }
+
+  void _addSelectEvent(List<Event> events) {
+    setState(() {
+      selectedEvents.addAll(events);
+    });
```

Comment:
still need setState ?

Reply:
try ListNotifier

Reply:
In my PR don't focus on refactor `selectedEvents `. I only focus change reply event. Improve this later